### ISSUE TITLE
Determine a Fedora Upgrade Path for AVR on AWS 

### ISSUE THREAD

**Issue by carrickr** [nulib/avalon#337]
The rails app itself is a no downtime issue, but we need to replace the Fedora:

**Get a Maintenance Window**

Get a window, probably an early AM one, and replace the Fedora instance.

**Build A New Stack**

Cloudform up a new AVR stack (with the new Fedora), copy solr data over to the new instance.  Test, then just swap stacks.  Plus side here is no maintenance window, downside is the need to make the old AVR read only after the solr copy, to avoid any kind of drift in data.

**Comment by mbklein** [comment]
I don't think we need to create a new stack. We have DB snapshots. We can snapshot the S3 bucket to glacier, swap in the new Fedora WAR, check it all over, and move on.
